code/game/mecha/mecha.dm

```diff
@@ -191,7 +191,7 @@
 
 /obj/mecha/proc/add_cabin()
 	cabin_air = new
-	cabin_air.temperature = T20C
+	cabin_air.temperature = T20C - 7

```

Comment:
As I asked in Discord, did you check if this is safe for a normal human and maybe Unathi?